source/azure_iot.c

```diff
@@ -0,0 +1,455 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot.c
+ *
+ * @brief Common Azure IoT code for FreeRTOS middleware.
+ *
+ */
+
+#include <stdio.h>
+
+#include "azure_iot.h"
+
+#define azureiotBASE64_HASH_BUFFER_SIZE    33
+
```

Comment:
please use ( 33 )

Reply:
Can we add comment that we are using SHA256 so we need 32 bytes

---
In reply to: [608240846](https://github.com/Azure/azure-iot-middleware-freertos/pull/58#discussion_r608240846) [](ancestors = 608240846)

Reply:
yes and yes